userGroupOption.xml

```diff
@@ -62,6 +62,14 @@
 				<maxvalue>100000</maxvalue>
 				<usersonly>1</usersonly>
 			</option>
+			<option name="user.conversation.maxStartedConversationsPer24Hours">
+				<categoryname>user.conversation</categoryname>
+				<optiontype>infiniteInteger</optiontype>
+				<defaultvalue>10</defaultvalue>
```

Comment:
50 might be a better default value for larger boards, without changing much for smaller ones where even 10 might include “almost all” users. I don't expect 50 spam emails to cause large issues with email reputation.

```suggestion
				<defaultvalue>50</defaultvalue>
```

Reply:
We'll add an explanation in the announcement that will inform of this change. While I do agree that "50" might be a sane value for large communities, I don't see this being necessary for the majority, including smaller communities. Site administrators can change this value at any time and since this is another line of defense against an actual threat, I'll lean on the "better safe than sorry" side.

Reply:
Fair enough.